README.md

```diff
@@ -29,7 +29,14 @@ To create new example:
 
 ## Documentation
 
-When we write documentation, we'll put links to it here.
+We have documentation in `/docs` that is [hosted using GitHub Pages](https://department-of-veterans-affairs.github.io/formulate/).
+
+To run the Jekyll site locally, run the following commands:
+
+```shell
+cd docs
+bundle exec jekyll serve
+```
```

Comment:
Looks like this works, but I'm not sure if the links are borked because of a Codespaces thing or what.

![image](https://user-images.githubusercontent.com/12970166/131559672-a6921921-8e09-41f6-8ac0-63e5d870c307.png)